platforms/hyperledger-indy/configuration/README.md

```diff
@@ -1,45 +1,56 @@
+- Use the 'BAF Build container' to create a containerized Ansible controller from which to deploy/manage your networks. This is recommended for development instances, as it is an easy way to build the required base environment for BAF deployment. More info can be found [here](https://blockchain-automation-framework.readthedocs.io/en/latest/developer/docker-build.html).
 
-## Prerequisites
+## Installation pre-requisites
+BAF requires tools such as Kubernetes, Git (repository), Vault and more to be installed.
+For more information on the installation pre-requisites, please refer to [this guide](https://blockchain-automation-framework.readthedocs.io/en/latest/prerequisites.html).
 
-To run the playbooks, following are the pre-requisites.
-1. Ansible 2.8.2 with jmespath installed (a docker image can be made from this [Dockerfile](../../shared/images/ansibleSlave.Dockerfile)).
-2. Ansible controller configured like this [sample](../../shared/inventory) inventory settings.
-3. One Managed Kubernetes cluster for each organization (currently Indy is only tested on Kubernetes 1.16 and also you need **AWS cli** credentials).
-4. A Hashicorp Vault installation for each organization which is initiated and unsealed. The Vault IP Address should be accessible from this machine (where the playbook is run), and the Vault root token is available.
-5. A Git User with write access to all the branches in the chosen Git repository. This user should be a service user and not a federated user.
-6. Edited and saved network.yaml with all fields populated according to your requirements. A Sample network.yaml can be found [here](./samples/network-indyv3.yaml)
+## Configuration pre-requisites
+For each organization in the DLT network you need to set up the following:
+1. One Managed Kubernetes cluster; BAF is currently tested on Amazon EKS, which means you will need AWS CLI set up as well.
+2. A Hashicorp Vault installation for each organization which is initialized and unsealed. The Vault Address should be accessible from this machine (where the playbook is run) and the Kubernetes cluster. The Vault root token is used in the network configuration, so this should be available as well.
+3. A Git User with write access to all the branches in the chosen Git repository; as well as an access token.
+4. The network configuration file (`network.yaml`) which has been filled in according to your requirements. A sample `network.yaml` for Hyperledger Indy can be found in [this folder](./samples/).
 
-## Execution
-
-**NOTE**: If you are running the playbooks independently i.e. not using platforms/shared/configuration/site.yaml, then use the same network.yaml as input for each playbook.
+For other general pre-quisites, such as Docker images, Ambassador and DNS setup, please refer to the ['Configure Pre-requisites' guide](https://blockchain-automation-framework.readthedocs.io/en/latest/operations/configure_prerequisites.html).\
```

Comment:
remove \

Reply:
Resolved in [a9c1489](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/a9c14893b67ea6f9eb1ef2e8a011949764891e53)